http/cars.html

```diff
@@ -1,21 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Hi,

glad you find some time to have a closer look.

The old cars.html example will work as well with the changes herein. But if it is to try the other approach I can put it back in.
I just created the cars.lua since I was too lazy to change the sources every time I want to test a different number of cars. It generates pretty much the same page as the cars.html.

As to the big PR:
I developed offline and then tried to separate the chenges inseparate checkins. They build up on each other. Cause of that it would be quite difficult to extract each feature/fix in a separate PR.
Id like to avoid that.